### DISCUSSION TITLE
Direction

### DISCUSSION THREAD

**Discussion by majidys** [requarks/wiki#5076] [General]
We really need the following feature :
direction: rtl and ltr
Please add to the editor

![image](https://user-images.githubusercontent.com/99164978/156871259-10cfdd52-c333-4ac1-aed3-0ca44141d0a1.png)

**Comment by majidys** [comment]
@johnnyshields @nilium @jforman @axisofentropy

**Reply by nilium** [reply]
Hey @majidys -- I agree that RTL support is especially important for ensuring folks can write in a wide range of languages, so hopefully this gets considered at some point (if it's not already supported). That said, I'm not sure why I'm being tagged here. I haven't done anything with Wiki.js in around two years (previous job decided not to use it), so I haven't looked at it in a fair amount of time. If you could avoid tagging me into feature requests in the future, I would appreciate it.

**Comment by majidys** [comment]
PLZ Help
How can I add this option?

**Comment by majidys** [comment]
https://ckeditor.com/docs/ckeditor5/latest/features/language.html
